createdIndex and modifiedIndex are not present on the root node

mehcode opened the pull request:
When getting the root node `/`, `created_index` and `modified_index` are not present.

---

Perhaps a better abstraction for this type is needed when all fields are optional. 😄

jimmycuadra commented:
Ugh, frustrating. In the previous release I had them marked as optional but from what I could tell of the Go source code they were always present. This is why I hate Go's type system and projects with no documentation. :{

Would you mind updating the tests to unwrap these options so the build will pass?

mehcode commented:
No problem. Fixed the tests and added an explicit test for what I fixed.

---

Nice library by the way. I just finished adding support for etcd using your library over in [config-rs](https://github.com/mehcode/config-rs/blob/etcd/examples/etcd/src/main.rs).

The only thing that confused me was when I accidentally sent an illegal path the library gave a serialization error and as I was trying to test behaviour if a key wasn't found, it seemed half-right and half-wrong. May I suggest a path check?

jimmycuadra commented:
One test failure, but I ran it locally and it passes. Seems to be an intermittent failure and it was already happening on previous builds, so I can merge this. Thanks much for the contribution and updating the tests. I'll publish as 0.8.0.

Can you open a separate issue for the path problem, perhaps with a failing test case?
